src/main/java/com/jland/website/service/impl/ConferenceServiceImpl.java

```diff
@@ -16,6 +16,15 @@ public ConferenceServiceImpl(ConferenceRepository conferenceRepository) {
 
     @Override
     public Conference getById(Long id) {
-        return conferenceRepository.getById(id);
+
+        return conferenceRepository.getById(id).orElse(null);
```

Comment:
I expect some exception like:
Conference with id 5 is not found

Reply:
added